src/v/cluster/security_manager.h

```diff
@@ -35,8 +35,8 @@ class security_manager final {
     }
 
 private:
-    template<typename Cmd>
-    ss::future<std::error_code> dispatch_updates_to_cores(Cmd);
+    template<typename Cmd, typename T>
```

Comment:
should this be name `typename Service` instead of `T` so you have commands and services.

Reply:
I'd also move the commit as a comment above here.

Reply:
fixed